transcom/mymove
# B 21557 INT find Estimated price when shipment is updated with an estimated weight where there isn't one previously

## [B-21557](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)
## Summary

Updates the price estimates on service items where an estimated weight wasn't initially provided.

### How to test

0. run `make tasks_save_ghc_fuel_price_data` so that the FSC service items will have the prerequisite data needed to calculate pricing on these items.
1. Create a move as an HHG shipment, or find an existing move that is already approved
2. As a TOO add an HHG shipment to the move. but do not add an estimated weight
3. Verify on the move task order tab that the service items are added, leave the tab open
4. Open a separate tab and find the move as a prime user
5. update the new mto shipment within the move with an estimated weight
6. back on the SC tab, refresh the mto shipment tab, the prices should now show next to the service items
